backend/typescript/services/implementations/camperService.ts

```diff
@@ -13,6 +13,20 @@ class CamperService implements ICamperService {
     let newCamper: Camper;
     let existingCamp: Camp | null;
     try {
+      existingCamp = await MgCamp.findById(camper.camp);
+
+      if (existingCamp) {
+        const existingCampObj = existingCamp.toObject();
```

Comment:
why did you need this `existingCampObj`, were you getting an error? I just tested your code using only existingCamp and it worked fine - I was getting this error from this line: "Property 'toObject' does not exist on type 'Camp'.ts(2339)"

Reply:
Yea I couldn't properly access the capacity property unless I cast the existing camp into an object --> now I see the same error as you so I'll remove it